src/components/publiq-ui/Container.js

```diff
@@ -0,0 +1,29 @@
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const StyledDiv = styled.div`
+  @media (min-width: 768px) {
+    width: 750px;
+    max-width: 750px;
+  }
+
+  @media (min-width: 992px) {
+    width: 970px;
+    max-width: 970px;
+  }
+
+  @media (min-width: 1200px) {
+    width: 1170px;
+    max-width: 1170px;
+  }
+`;
+
+const Container = ({ children }) => {
```

Comment:
Make sure we always pass down className too. If we would like to wrap this component with a styled() again, it needs access to the className.